common/services/certificate.js

```diff
@@ -22,6 +58,7 @@ const storeCert = function (cert) {
 }
 
 const deleteCert = async function() {
+  localStorage.removeItem(CERT);
   await IDB.deleteDatabase();
 }
```

Comment:
Only thing I'm unsure of: does calling this also delete the private key used to request the cert in the first place? If you use the same private key for renewing the certificate, you should be able to do so without triggering 2FA.

Reply:
do you mean the **public** key?
![image](https://github.com/doordeck/doordeck-sdk-javascript/assets/6842786/29b01c36-6747-4b6a-b3a7-35a05b7bdefb)

When a certificate is deleted for some reason, it calls for a new one and it doesn't trigger 2FA, which seemed good to me AFAIK

Reply:
Well, I mean both public/private keys, but obviously you only send the public key to Doordeck, but yeah so long as the ephemeral key isn't deleted this sounds reasonable.

Reply:
I'll double check again after merging because you made me doubt 👀 but yep as long as I'm not getting asked for 2FA these keys wouldn't get deleted

Reply:
* before merging

Reply:
exactly, keys are saved under the following 2 string keys: `EPHEMERAL_PRIKEY` and `EPHEMERAL_PUBKEY`:

<img width="1024" alt="image" src="https://github.com/doordeck/doordeck-sdk-javascript/assets/6842786/860043a3-8f00-4dd1-9973-09f8eecdba81">

so no damage